api/src/test/java/org/openmrs/api/ProgramWorkflowServiceTest.java

```diff
@@ -472,7 +476,40 @@ public void purgeProgram_shouldPurgeProgramWithPatientsEnrolled() {
 		// make sure that the program was deleted properly
 		assertNull(Context.getProgramWorkflowService().getProgram(2));
 	}
-	
+	@Test
+	public void shouldTestGetAllProgramAttributeTypes() throws Exception {
+	assertEquals(1, pws.getAllProgramAttributeTypes().size());
```

Comment:
It would not hurt to indent the above. 😊

Reply:
sorry, my bad.